### DISCUSSION TITLE
MIT License Option

### DISCUSSION THREAD

**Discussion by TCROC** [StardustDL/Linq-in-Rust#8] [General]
Would you consider adding an optional MIT license so that people can opt to use MIT or Apache at their choice?  I've seen several repos do this including the main rust repo: https://github.com/rust-lang/rust

I personally find the license much more friendly and appealing than the Apache license.

**Comment by StardustDL** [comment]
Recently I have no plan to change the license, but I will consider this : )

**Comment by TCROC** [comment]
Awesome thanks!  What is the reason for choosing Apache if I may ask?  I'm not super familiar with the benefits of it compared to MIT.

**Comment by StardustDL** [comment]
I think Apache keeps a little limitation and it is friendly enough when creating this project, so I choose it. I'm not evaluated the license much since I'm also not really familiar with this 😂.

**Comment by TCROC** [comment]
Sounds good! Thanks for the responses! 😄 Should we keep this issue open for potential future discussion then?
